src/main.ts

```diff
@@ -26,6 +29,7 @@ export function configure(aurelia: Aurelia): void {
       configuration.settings.keyboard = false;
     });
   aurelia.use.singleton(HTMLSanitizer, DOMPurify);
+  aurelia.use.singleton(DataSourceDeals, CeramicServiceMock);
```

Comment:
@dkent600 
#### Use implementation as "key"
In the pair session, we had
```suggestion
  aurelia.use.singleton(IDataSource, CeramicServiceMock);
```

But this gave an DI error: 
`Error: key/value cannot be null or undefined. Are you trying to inject/register something that doesn't exist with DI?`

So I had to go the way to create an Implementation `IDataSource` in `DataSourceDeals.ts`

#### Other approaches
I tried other approaches with no luck

```ts
export declare class IDataSourceDeals;

export declare class DataSourceDeals extends IDataSourceDeals {}

export declare class DataSourceDeals implements IDataSourceDeals {
  initialize(rootId?: string): void;
  get<T>(id?: string): T;
  create(registration: IDealConfig): Promise<string>;
  save(id: string, registration: IDealConfig): void;
  update(id: string, registration: IDealConfig): void;
}
```